yelp_package/dockerfiles/focal/Dockerfile

```
@@ -21,8 +21,7 @@ RUN rm /etc/dpkg/dpkg.cfg.d/excludes
 RUN apt-get update && apt-get install -yq gnupg2
 
 RUN apt-get update > /dev/null && \
-    apt-get install -y --no-install-recommends software-properties-common && \
-    add-apt-repository ppa:deadsnakes/ppa
+    apt-get install -y --no-install-recommends software-properties-common
```

```suggestion
RUN apt-get update > /dev/null &&
```
Don't need s-p-c if not installing a PPA